TL improvement (#523)
07th-mod/higurashi-console-arcs: Update/omot_002.txt

//rだが！　この鬼のように猛り狂った空腹の前には、この程度の量など無力というもの！　俺のハングリーハートをもってすれば秒殺さえ可能だ！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だが！　この鬼のように猛り狂った空腹の前には、この程度の量など無力というもの！　俺のハングリーハートをもってすれば秒殺さえ可能だ！",

> 			NULL, "Still! Even a portion this size would be powerless before me with the appetite of a ferocious demon! With my hungry heart I could kill this in seconds!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Still! Even a portion this size would be powerless before my ferocious demon-like appetite! With my hungry heart I could kill this in seconds!", Line_Normal);
```

//r一方で沙都子は、ムンクの叫びのように真っ青な顔と涙目でイヤイヤと首を振り続けた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "一方で沙都子は、ムンクの叫びのように真っ青な顔と涙目でイヤイヤと首を振り続けた。",

> 			NULL, "Meanwhile, Satoko was shaking and crying out, blue in the face with tears in her eyes like the Scream.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Meanwhile, Satoko was shaking and crying out, blue in the face with tears in her eyes like in The Scream.", Line_Normal);
```

//r魅音もレナも、そろってbしたつづみ.<舌鼓>を打つ。たかがカボチャの煮転がしと侮るなかれ、ダシの漬け込み具合がほどよくなければ、この食感と味は表現できまい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "魅音もレナも、そろって舌鼓(したつづみ)を打つ。たかがカボチャの煮転がしと侮るなかれ、ダシの漬け込み具合がほどよくなければ、この食感と味は表現できまい。",

> 			NULL, "Mion and Rena both smacked their lips together. Don't see this as a mere boiled and mashed squash. Without properly pickling squash in dashi, you can't make it into this taste and texture.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mion and Rena both smacked their lips together. Don't see this as merely boiled and mashed squash. Without properly pickling squash in dashi, you can't produce this taste and texture.", Line_Normal);
```

//rしかしそもそも、ただでさえアウトローちっくな詩音には最も効き目のない理屈だったようで、反対に鼻で笑われ『ひっ？！』と身をすくませた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "しかしそもそも、ただでさえアウトローちっくな詩音には最も効き目のない理屈だったようで、反対に鼻で笑われ『ひっ？！』と身をすくませた。",

> 			NULL, "But of course, that line of reasoning was ineffective against an outlaw like Shion. So she snorted at Satoko with laughter and made her shrink back.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But of course, that line of reasoning was ineffective against an outlaw like Shion. So she scornfully laughed at Satoko and made her shrink back.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300024", 256, TRUE);
	OutputLine(NULL, "「ちょ、超法規的措置を使うとは…っ！」",

> 			NULL, "\"Th-that's extra-legal...!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Th-that's completely illegal...!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "いや…その責任感は賞賛に値すると思うが、そこまで悟史は頼んでいたのだろうか？　激しく首を振って否定している、顔も見たこともない沙都子の兄が一瞬脳裏に浮かんだ。",

> 			NULL, "Well... I do think that level of responsibility deserves praise, but did Satoshi really ask you to go this far? I could briefly imagine Satoko's older brother who I'd never even met before was violently shaking his head to chase off the thought.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Well... I do think that level of responsibility deserves praise, but did Satoshi really ask you to go this far? Though I'd never met Satoko's older brother before, I could briefly imagine him violently shaking his head to chase off the thought.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "それは一種、異様な光景だった。もはや敗色濃厚だった沙都子が缶詰を水戸黄門の印篭のようにかざすと、あれほどに圧倒的だった詩音が一歩、また一歩と、じりじりと気圧されている…。",

> 			NULL, "It was a totally one-of-a-kind-scene. Satoko had finally regained her composure, and once she showed off the can like a police badge, the normally high-handed Shion took one step back after another, overwhelmed...", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was a totally one-of-a-kind-scene. Satoko had finally regained her composure, and once she held up the can like a police badge, the normally high-handed Shion took one step back after another, overwhelmed...", Line_Normal);
```

//rその言葉を聞いた瞬間、『ぴくっ！』と跳ねるように詩音の肩が動いた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その言葉を聞いた瞬間、『ぴくっ！』と跳ねるように詩音の肩が動いた。",

> 			NULL, "As soon as Shion heard that, she gave a jump with her shoulders slightly moving.", Line_Normal);

Reviewer: ```suggestion
			NULL, "As soon as Shion heard that, her shoulders twitched like she recoiled back.", Line_Normal);
```
